servant-docs/test/Servant/DocsSpec.hs

```diff
@@ -126,8 +126,10 @@ spec = describe "Servant.Docs" $ do
     it "mentions headers" $ do
       md `shouldContain` "- This endpoint is sensitive to the value of the **X-Test** HTTP header."
 
-    it "contains response samples" $
-      md `shouldContain` "{\"dt1field1\":\"field 1\",\"dt1field2\":13}"
+    it "contains response samples - dt1field1" $
```

Comment:
Thanks, we merged your fix for the cookbook as part of #1397 Could you remove it from your branch and rebase on master ?

Reply:
Rebased! And the build is passing.